src/openzaak/config/forms.py

```diff
@@ -43,3 +48,41 @@ def __init__(self, *args, **kwargs):
 
         if instance and instance.component == "autorisaties":
             self.fields["enabled"].disabled = True
+
+
+def get_nlx_choices() -> Dict[str, dict]:
+    choices = {}
+    nlx_outway = NLXConfig.get_solo().outway
+    try:
+        services = get_services()
+    except Exception:
+        return {}
+
+    for service in services:
+        url = f"{nlx_outway}{service['organization_name']}/{service['service_name']}/"
```

Comment:
note that NLX is introducing changes to outway URLs: https://gitlab.com/commonground/nlx/nlx/-/issues/530